packages/esm-patient-clinical-view-app/src/clinical-view-overview/clinical-view-overview.component.tsx

```diff
@@ -23,6 +24,7 @@ const ClinicalViewOverview: React.FC<ClinicalViewOverviewProps> = ({ patientUuid
   }, [patient, patientUuid]);
 
   const launchClinicalViewForm = React.useCallback(() => {
+    clearWorkspace('patient-chart-workspace-slot');
```

Comment:
@walteronoh am not sure if this is the best approach to solve this. What happens to forms that have been minimised in the workspace or forms with pending data.